TagshelfApi/Client/Concrete/TagshelfApiClient.cs

```diff
@@ -0,0 +1,56 @@
+﻿using RestSharp;
+using RestSharp.Deserializers;
+using TagshelfApi.Cache.Abstract;
+using TagshelfApi.Client.Abstract;
+using TagshelfApi.Responses;
+using TagshelfApi.Tooling.Abstract;
+
+namespace TagshelfApi.Client.Concrete
+{
+    public class TagshelfApiClient : BaseClient
+    {        
+        public TagshelfApiClient(string apiKey,
```

Comment:
This will be refactored later to actually be able to support different authentication methods.